2021/Application.md

```diff
@@ -0,0 +1,68 @@
+# Application process
+Technical writers interested in participating to GSoD with ArviZ should follow these steps:
+
+1. Join the dedicated ArviZ Gitter channel for season of docs: https://gitter.im/arviz-devs/season_of_docs
+2. Send an statement of interest to `arviz.devs@gmail.com` before May 1st. More info about your statement of
+   interest at []().
+3. Fulfill the application requirements before May 10th.
+4. Be available for feedback on your proposal.
+
+## Application requirements
+Only applications from people who perform the couple tasks below will be considered.
+
+The only requirement will be ...
+
+---
+---
+
+# Selection criteria
```

Comment:
Should we add general expectations her for project? Like we imagine a project will include user study, docs study overview, specific suggestions ohn how to organize etc etc?

Reply:
I would add that in its own file, the detailed scope and priorities I mentioned above

Reply:
Where above?

Reply:
> We should also add a "detailed scope and priorities" doc, where we extend https://github.com/arviz-devs/arviz/wiki/Season-of-Docs-2021#projects-scope with everything we have written in Drive and Slack.

I'll try to draft something like this and push in ~20 mins maybe?

Reply:
as it's a new file there should not be merge issues so you can keep working on these 2 docs

Reply:
draft pushed, still needs work though